### DISCUSSION TITLE
Z2M is writing last restart time to genTime cluster instead of current

### DISCUSSION THREAD

**Discussion by lacrimal** [Koenkk/zigbee2mqtt#21010] [General]
Hi,
It is not necessary technical issue - if yes - i will make issue report.
I have 3 POOP TRV, after z2m restart there is message generated for them to set genTime cluster like below.

{"timeStatus":1,"dstShift":3600,"timeZone":3600,"dstStart":765162000,"dstEnd":783302400,"time":759246239}' to 'genTime'

However it looks like time itself is not fetched from some kind of 'now' function and it is set as previous restart of z2m was done.
I have whole installation dockerized, because it is restartet sporadicaly - not seen this issue before. 
I have this weird behavior on latest and latest-dev versions of zigbee2mqtt docker and it was definitely not a problem 8 months ago.

Can you guide me to resolve this - maybe it is simple thing.

**Comment by Koenkk** [comment]
I'm not sure why this happens, the current date is taken [here](https://github.com/Koenkk/zigbee-herdsman/blob/master/src/controller/model/device.ts#L240)
